include/gsl/span

```diff
@@ -361,13 +361,13 @@ public:
 
     template <std::size_t N, class ArrayElementType = std::remove_const_t<element_type>>
     constexpr span(std::array<ArrayElementType, N>& arr) noexcept
-        : storage_(&arr[0], details::extent_type<N>())
+        : storage_(arr.data(), details::extent_type<N>())
```

Comment:
Would std::data(arr) work as well? From memory, that is closer to the WD description of what happens.

Reply:
std::data would work too, but it was introduced in C++17. GSL generally assumes C++14 right?